brave/brave-core
# Fixes grant locale mix

Resolves https://github.com/brave/brave-browser/issues/2945

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests`) on

## Test Plan:
- enable rewards
- go to panel and make sure that title doesn't have claim inside
- go to rewards summary (panel and settings) and make sure that claim is there

## Reviewer Checklist:
